cloudhub-2/modules/ROOT/pages/ch2-configure-horizontal-autoscaling.adoc

```diff
@@ -0,0 +1,138 @@
+
+The CPU-based HPA policy used for all Mule apps deployed to CloudHub 2.0 is as follows:
+
+----
+apiVersion: autoscaling/v2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: my-app
+  namespace: app-namespace
+spec:
+  behavior:
+    scaleDown:
+      policies:
+      - periodSeconds: 15
+        type: Percent
+        value: 100
+      selectPolicy: Max
+      stabilizationWindowSeconds: 1800
+    scaleUp:
+      policies:
+      - periodSeconds: 180
+        type: Percent
+        value: 100
+      selectPolicy: Max
+      stabilizationWindowSeconds: 0
+  maxReplicas: 3
+  metrics:
+  - resource:
+      name: cpu
+      target:
+        averageUtilization: 70
+        type: Utilization
+    type: Resource
+  minReplicas: 1
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: my-app
+----
+
+Some points to consider:
+
+Scale up can occur at most every 180 seconds. Each period, up to 100% of the currently running replicas may be added until the maximum configured replicas is reached. For scaling up there is no stabilization window. When the metrics indicate that the target should be scaled up, the target is scaled up immediately.
+
+Scale down can occur at most every 15 seconds. Each Period, up to 100% of the currently running replicas may be removed which means the scaling target can be scaled down to the minimum allowed replicas. The number of replicas removed is based on the aggregated calculations over the past 300 seconds of the stabilization window.
+
+Min replicas +
+
+* The minimum number of replicas that would be guaranteed to run at any given point of time.
+* Scale down policy would never remove replicas below this number.
+
+Max replicas +
+
+* The maximum number of replicas that are capped, beyond which no more replicas can be added for scale up.
+* Scale up policy would never add replicas above this number.
+
+
+== Performance Considerations and Limitations
+
+For a successful horizontal autoscaling of your Mule apps, review the following performance considerations:
+
+* Mule apps that scale based on CPU usage are a good fit with CPU based HPA. For example: 
+** HTTP/HTTPS applications with async requests.
+** Reverse Proxies.
+** Low latency+High throughput applications.
+** Dataweave Transformations.
+** APIKit Routing.
+** API Gateways with policies.
+* Non-reentrant applications that do not have built-in parallel processing such as batch jobs, scheduler applications without re-entrancy and duplicate scheduling across applications, and low-throughput, high-latency applications with large requests are not a good fit with CPU-based HPA.
+
+=== Limitations
+
+There are some limitations to consider:
+
+* CPU-based HPA works for micro and micro memory-intensive xref:general::pricing.adoc#cloudhub-2-0-application-deployment[replica sizes] for customers on Integration - Starter.
```

Comment:
Missing that Integration - Advanced customers can also use 'Small' replica sizes. 

also instead of having these 2 under limitations section, let us move it under the Configure Horizontal Pod Autoscaling section (as mentioned in another comment). We can remove the "Limitations" section then.